packages/nimbus/src/components/collapsible-motion/collapsible-motion.tsx

```diff
@@ -0,0 +1,16 @@
+import { CollapsibleMotionRoot } from "./components/collapsible-motion-root";
+import { CollapsibleMotionTrigger } from "./components/collapsible-motion-trigger";
+import { CollapsibleMotionContent } from "./components/collapsible-motion-content";
+
+// Exports for internal use by react-docgen
+export {
+  CollapsibleMotionRoot as _CollapsibleMotionRoot,
+  CollapsibleMotionTrigger as _CollapsibleMotionTrigger,
```

Comment:
The UI Kit [collapsible motion component](https://github.com/commercetools/ui-kit/blob/main/packages/components/collapsible-motion/src/collapsible-motion.tsx#L0-L1) only handles setting enter/exit animations on the "panel"/"content", it doesn't concern itself with the `trigger` at all.

As an animation util, I think this makes sense because there are use cases where components need to be animated on enter, but the event that controls the display/hide is not directly controlled by the user.

Are these components necessary, or could we simplify things to match the UI Kit version?

Reply:
UI-Kit's components render props exposing `toggle` kind of implicitly makes it concerned with colocating a trigger element though, right?

We don't actually _need_ the trigger in the Nimbus component (there's a controlled trigger-less story that covers that), it's really just a bonus to make the consumer's life easier.